src/db/schema.ts

```diff
@@ -62,4 +62,17 @@ export async function initGatewayDb(db: Knex) {
       table.string("project", 64).index();
     });
   }
+
+  if (!(await db.schema.hasTable("stats"))) {
+    await db.schema.createTable("stats", (table) => {
+      table.string("contract_id", 64).primary();
+      table.string("owner", 64).index();
+      table.bigInteger("block_height").notNullable().index();      
+      table.string("block_id").notNullable();
+      table.string("timestamp").index();
+      table.jsonb("tags");
```

Comment:
shouldn't tags be a separate table? I believe it could simplify future queries (I know that postgres has cool support for json, but still..)

Reply:
also, table should be named `stats_contracts` - and if we decide to move the tags to a separate one - `stats_tags`